1436790/1438462: Part 12: 15 files with MSDN link fixes.

DennisLee-DennisLee opened the pull request:
Contributes to PR #[2037](https://github.com/dotnet/docs/issues/2037) and applies to the following work items:
- [User Story 1436790: Content Update: update MSDN links for .NET content](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1436790)
- [Task 1438462: Update MSDN links for .NET content](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1438462)

DennisLee-DennisLee committed:
20 files with MSDN link fixes.

DennisLee-DennisLee committed:
Found the repo for a previous-version link.

mairaw commented on docs/framework/data/adonet/ef/performance-considerations.md:
```diff
@@ -110,15 +110,15 @@ This topic describes performance characteristics of the ADO.NET Entity Framework
   
 #### Pre-generate views  
- Generating views based on an entity model is a significant cost the first time that an application executes a query. Use the EdmGen.exe utility to pre-generate views as a Visual Basic or C# code file that can be added to the project during design. You could also use the Text Template Transformation Toolkit to generate pre-compiled views. Pre-generated views are validated at runtime to ensure tha ...
+ Generating views based on an entity model is a significant cost the first time that an application executes a query. Use the EdmGen.exe utility to pre-generate views as a Visual Basic or C# code file that can be added to the project during design. You could also use the Text Template Transformation Toolkit to generate pre-compiled views. Pre-generated views are validated at runtime to ensure tha ...
```
```suggestion
 Generating views based on an entity model is a significant cost the first time that an application executes a query. Use the EdmGen.exe utility to pre-generate views as a Visual Basic or C# code file that can be added to the project during design. You could also use the Text Template Transformation Toolkit to generate pre-compiled views. Pre-generated views are validated at runtime to ensure that they are consistent with the current version of the specified entity model. For more information, see [How to: Pre-Generate Views to Improve Query Performance](https://docs.microsoft.com/previous-versions/dotnet/netframework-4.0/bb896240(v=vs.100)).
```

mairaw commented on docs/framework/data/adonet/ef/provider-manifest-specification.md:
```diff
@@ -85,7 +85,7 @@ public DbProviderManifest GetProviderManifest(string manifestToken);
   
 #### Using a Provider Manifest Token  
- For the offline scenario, the token is picked from SSDL representation. The SSDL allows you to specify a ProviderManifestToken (see [Schema Element (SSDL)](https://msdn.microsoft.com/library/fec75ae4-7f16-4421-9265-9dac61509222) for more information). For example, if a connection cannot be opened, the SSDL has a provider manifest token that specifies information about the manifest.  
+ For the offline scenario, the token is picked from SSDL representation. The SSDL allows you to specify a ProviderManifestToken (see [Schema Element (SSDL)](https://docs.microsoft.com/previous-versions/dotnet/netframework-4.0/bb387166(v=vs.100)) for more information). For example, if a connection cannot be opened, the SSDL has a provider manifest token that specifies information about the manifes ...
```
```suggestion
 For the offline scenario, the token is picked from SSDL representation. The SSDL allows you to specify a ProviderManifestToken (see [Schema Element (SSDL)](/ef/ef6/modeling/designer/advanced/edmx/ssdl-spec#schema-element-ssdl) for more information). For example, if a connection cannot be opened, the SSDL has a provider manifest token that specifies information about the manifest.  
```

mairaw commented on docs/framework/data/adonet/ef/sqlclient-for-the-entity-framework.md:
```diff
@@ -12,7 +12,7 @@ This section describes the .NET Framework Data Provider for SQL Server (SqlClien
   
 ## ProviderManifestToken Schema Attribute  
- `ProviderManifestToken` is a required attribute of the `Schema` element in SSDL. This token is used to load the provider manifest for offline scenarios. For more information about `ProviderManifestToken` attribute, see [Schema Element (SSDL)](https://msdn.microsoft.com/library/fec75ae4-7f16-4421-9265-9dac61509222).  
+ `ProviderManifestToken` is a required attribute of the `Schema` element in SSDL. This token is used to load the provider manifest for offline scenarios. For more information about `ProviderManifestToken` attribute, see [Schema Element (SSDL)](https://docs.microsoft.com/previous-versions/dotnet/netframework-4.0/bb387166(v=vs.100)).  
```
```suggestion
 `ProviderManifestToken` is a required attribute of the `Schema` element in SSDL. This token is used to load the provider manifest for offline scenarios. For more information about `ProviderManifestToken` attribute, see [Schema Element (SSDL)](/ef/ef6/modeling/designer/advanced/edmx/ssdl-spec#schema-element-ssdl).  
```

mairaw commented on docs/framework/data/adonet/ef/terminology.md:
```diff
@@ -8,61 +8,61 @@ This topic defines terms frequently referenced in [!INCLUDE[adonet_ef](../../../
 |----------|----------------|  
-|association|The definition of a relationship between entity types.<br /><br /> For more information, see [Association Element (CSDL)](https://msdn.microsoft.com/library/c305169a-8af7-432f-9ba7-800a163aed41) and [association type](../../../../../docs/framework/data/adonet/association-type.md).|  
-|association set|A logical container for instances of associations of the same type.<br /><br /> For more information, see [AssociationSet Element (CSDL)](https://msdn.microsoft.com/library/512cbb75-cebe-4f3f-970d-3419deeff684) and [association set](../../../../../docs/framework/data/adonet/association-set.md).|  
+|association|The definition of a relationship between entity types.<br /><br /> For more information, see [Association Element (CSDL)](https://docs.microsoft.com/previous-versions/dotnet/netframework-4.0/bb399734(v=vs.100)) and [association type](../../../../../docs/framework/data/adonet/association-type.md).|  
```
can you review this topic to make sure we're linking to the existing subsections in https://docs.microsoft.com/en-us/ef/ef6/modeling/designer/advanced/edmx/csdl-spec when available?

mairaw reviewed (changes requested):
Looks good @DennisLee-DennisLee. As usual, see my comments before we merge this.

DennisLee-DennisLee committed:
Merged from master. Fixed a conflict with migration-considerations.md (accepted incoming change).

mairaw committed:
Apply suggestions from code review

Accepting Maira's committable suggestions. Will review the rest after PR #10330 is merged.

Co-Authored-By: DennisLee-DennisLee <v-dele@microsoft.com>

mairaw commented:
@DennisLee-DennisLee now that I merged Part 11, we got merge conflicts. Can you resolve them? Thanks.

DennisLee-DennisLee committed:
Changed links to CSDL elements from "previous-version" URLs to subsection targets within "CSDL Specification".

DennisLee-DennisLee committed:
Fixed merge conflicts by accepting incoming changes (from part 11 - PR#10330).

terminology.md needed a manual change after fixing the conflict, though.

DennisLee-DennisLee committed:
Fixed multiple formatting issues.

DennisLee-DennisLee committed:
Final manual fix out of Maira's suggestions.

DennisLee-DennisLee commented:
@mairaw I think I've addressed everything in this and part 13 (PR #10351).

mairaw reviewed (approved):
Thanks @DennisLee-DennisLee! 👍
